gen/src/write.rs

```diff
@@ -1352,6 +1352,7 @@ fn write_space_after_type(out: &mut OutFile, ty: &Type) {
 enum UniquePtr<'a> {
     Ident(&'a Ident),
     CxxVector(&'a Ident),
+    CxxVectorPtr(&'a Ident, PtrConstness),
```

Comment:
The mutability should probably go first 
```suggestion
    CxxVectorPtr(PtrConstness, &'a Ident),
```